torch_nets/models.py

```diff
@@ -135,5 +135,5 @@ def endow_with_netcdf_params(model: nn.Module, nc_file: str):
     data_set = nc.Dataset(nc_file)  # pylint: disable=no-member
 
     for i, layer in enumerate(model.layers):
-        layer.weight[:] = data_set[f"w{i+1}"]
-        layer.bias[:] = data_set[f"b{i+1}"]
+        layer.weight[:] = torch.tensor(data_set[f"w{i+1}"][:])
+        layer.bias[:] = torch.tensor(data_set[f"b{i+1}"][:])
```

Comment:
Please could you clarify what this commit did as I am still struggling to see without looking in depth.

Reply:
It was because `data_set[f"w{i+1}"]` was not of the numpy array data type. Appending `[:]` to it will get the numpy array data. Previously there was a type error since `torch.tensor` requires the argument to be a numpy array.